### PR TITLE
Temporary fix for test suite

### PR DESCRIPTION
**Pull request by mheon**
We leak open files when creating new c/storage stores (locks do not close themselves, so the open FDs in the test suite increase every time we use c/storage to load cached images for the tests). Fix this temporarily by increasing rlimits on open files until we can create a permanent fix next release.

### CODE REVIEW

**Comment by mheon** [comment]
@rhatdan @baude @TomSweeneyRedHat PTAL. Let's get this merged to unblock other PRs and cut a release.

File: test/e2e/libpod_suite_test.go [hunk 1]
```diff
@@ -337,6 +347,7 @@ func (p *PodmanTest) RestoreArtifact(image string) error {
 	}()
 
 	storage.Transport.SetStore(store)
+
```
[the remark this answers is not in the document]
Comment (mheon) [reply test/e2e/libpod_suite_test.go]: Yeah, the function was almost unreadable before

File: test/e2e/libpod_suite_test.go [hunk 2]
```diff
@@ -326,9 +339,6 @@ func (p *PodmanTest) RestoreArtifact(image string) error {
 	storeOptions.GraphRoot = p.CrioRoot
 	storeOptions.RunRoot = p.RunRoot
 	store, err := sstorage.GetStore(storeOptions)
-	defer store.Shutdown(false)
```
[the remark this answers is not in the document]
Comment (mheon) [reply test/e2e/libpod_suite_test.go]: There is actually another `defer` below that also shuts down the store

File: test/e2e/libpod_suite_test.go (hunk 1 above)
[the remark this answers is not in the document]
Comment (rhatdan) [reply test/e2e/libpod_suite_test.go]: ok

**Comment by rhatdan** [comment]
LGTM (Really looks bad to me, but I would rather get the feature in)>

**Comment by baude** [comment]
@rh-atomic-bot r+

**Pull request closed without merging**